posydon/popsyn/rate_calculation.py

```diff
@@ -31,11 +32,16 @@
     'Z_max' : 1.,
     'select_one_met' : False,
     'dlogZ' : None, # e.g, [np.log10(0.0142/2),np.log10(0.0142*2)]
-    'Zsun' : Zsun
+    'Zsun' : Zsun,
+    'compute_GRB_properties' : False,
+    'GRB_beaming' : None, # e.g., 0.5, 'Goldstein+15'
+    'GRB_efficiency' : None, # e.g., 0.01
+    'E_GRB_iso_min' : None, # e.g., 1e51 erg 
```

Comment:
Why do you use unsupported `None` for default values? I'd suggest to use
`'GRB_beaming' : 1.0,
'GRB_efficiency' : 0.0,
'E_GRB_iso_min' : 0.0,`